src/app/client/src/app/modules/search/components/course-search/course-search.component.ts

```diff
@@ -152,7 +152,7 @@ export class CourseSearchComponent implements OnInit, OnDestroy, AfterViewInit {
       return o.name === (selectedMediaType || 'all');
     });
     let filters = _.pickBy(this.queryParams, (value: Array<string> | string) => value && value.length);
-    filters = _.omit(filters, ['key', 'sort_by', 'sortType', 'appliedFilters', 'selectedTab', 'mediaType']);
+    filters = _.omit(filters, ['key', 'sort_by', 'sortType', 'appliedFilters', 'selectedTab', 'mediaType', 'utm_source']);
```

Comment:
Write unit test for this.
Check:  unt_course will never go in request payload at all.